src/composeapp.cc

```diff
@@ -86,6 +150,14 @@ bool ComposeApp::cmd_streaming(const std::string& cmd) {
   return boost::process::system(cmd, boost::process::start_dir = root_) == 0;
 }
 
+std::pair<bool, std::string> ComposeApp::cmd(const std::string& cmd) {
+  auto compose_file = root_ / ComposeFile;
+  std::string out_str;
+  int exit_code = Utils::shell(compose_ + " -f " + compose_file.string() + " " + cmd, &out_str);
```

Comment:
This should be OK since by default a project dir is equal to the Compose file directory, but I will double-check it.
`--project-directory PATH    Specify an alternate working directory
                              (default: the path of the Compose file)
`